tests/integration_tests/bugs/test_lp1897099.py

```diff
@@ -0,0 +1,39 @@
+""" Integration test for LP #187099
+
+Ensure that if fallocate fails during mkswap that we fall back to using dd
+
+https://bugs.launchpad.net/cloud-init/+bug/1897099
+"""
+
+import pytest
+
+
+USER_DATA = """\
+#cloud-config
+swap:
+  filename: /swap.img
+  size: 10000000
+  maxsize: 10000000
+"""
+
+
+@pytest.mark.user_data(USER_DATA)
+@pytest.mark.lxd_vm
```

Comment:
Heh...I already had a not_container mark (and whole separate instance_setup mark that could take a callback). This test wouldn't work on GCE and since the reboot thing isn't "normal" behavior I didn't spend much time investigating. I could maybe bring that mark back with the bootcmd way though